janus-pp-rec has no stdout

There is no stdout, when using janus-pp-rec:
1) Without options
2) With --header option

Janus from "master branch"
Cloned this time

Maybe this can help? 
```
externuz@mediacompiler:/opt/janus-gateway$ make
make  all-recursive
make[1]: Entering directory `/opt/janus-gateway'
Making all in html
make[2]: Entering directory `/opt/janus-gateway/html'
make[2]: Nothing to be done for `all'.
make[2]: Leaving directory `/opt/janus-gateway/html'
make[2]: Entering directory `/opt/janus-gateway'
  CC       postprocessing/janus_pp_rec-janus-pp-rec.o
postprocessing/janus-pp-rec.c: In function ‘main’:
postprocessing/janus-pp-rec.c:359:9: warning: ignoring return value of ‘fread’, declared with attribute warn_unused_result [-Wunused-result]
    fread(prebuffer, sizeof(char), 1, file);
         ^
  CCLD     janus-pp-rec
make[2]: Leaving directory `/opt/janus-gateway'
make[1]: Leaving directory `/opt/janus-gateway'
```

--- a/postprocessing/janus-pp-rec.c
+++ b/postprocessing/janus-pp-rec.c
@@ -76,6 +76,7 @@ void janus_pp_handle_signal(int signum) {
 int main(int argc, char *argv[])
 {
 	janus_log_init(FALSE, TRUE, NULL);
+	atexit(janus_log_destroy);
 
 	/* Check the JANUS_PPREC_DEBUG environment variable for the debugging level */
 	if(g_getenv("JANUS_PPREC_DEBUG") != NULL) {
@@ -519,8 +520,5 @@ int main(int argc, char *argv[])
 	}
 
 	JANUS_LOG(LOG_INFO, "Bye!\n");
-
-	janus_log_destroy();
-
 	return 0;
 }

